src/Library/demos/Row/main.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Rows";
+  description: "Create user interface lists with ease";
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+      Adw.PreferencesGroup  {
+
+      title: "Rows can be groupped";
+
+      Adw.ActionRow {
+        title: "Action row can have an icon";
+        icon-name: "info";
+      }
+
+      Adw.ActionRow {
+        title: "Action row can have a prefix child";
+
+        [prefix]
+        CheckButton checkbox {
+          active: true;
+          }
+      }
+
+      Adw.ActionRow {
+        title: "Action row can have a suffix child";
+        subtitle: "Select checkbox above";
+
+        [suffix]
+        Spinner {
+          spinning: bind checkbox.active;
+        }
+      }
+
+      Adw.EntryRow {
+         title: "A row can be an entry";
+      }
+    }
+
+    Adw.PreferencesGroup {
+      title: "Groups can have a suffix";
+
+      [header-suffix]
+      Adw.ButtonContent {
+      label: "Settings";
+      icon-name: "org.gnome.Settings-symbolic";
```

Comment:
According to Icon Library it's preinstalled with the os, but it seems it's not true

Reply:
Use the Icon Library in Workbench - it won't deceive you